Buổi 10/homework.js

```diff
@@ -0,0 +1,146 @@
+//Bài 1

```

Comment:
```suggestion
// Bài 1
```
Và luôn phải sử dụng tiếng anh, ngoại trừ được yêu cầu dùng ngôn ngữ khác, hoặc bắt buộc phải dùng ngôn ngữ khác